src/Sarif.Viewer.VisualStudio/CodeAnalysisResultManager.cs

```diff
@@ -46,6 +48,9 @@ internal sealed class CodeAnalysisResultManager : IVsSolutionEvents
 
         private readonly IFileSystem _fileSystem;
 
+        private readonly ConcurrentDictionary<string, ResolveEmbeddedFileDialogResult> userDialogPreference
+            = new ConcurrentDictionary<string, ResolveEmbeddedFileDialogResult>();
```

Comment:
>new ConcurrentDictionary<string, ResolveEmbeddedFileDialogResult>(); [](start = 14, length = 68)

since we have a constructor, lets use it. Move this to the constructor

Reply:
updated

---
In reply to: [606203534](https://github.com/microsoft/sarif-visualstudio-extension/pull/384#discussion_r606203534) [](ancestors = 606203534)